.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.linting.pylintEnabled": true,
+    "python.formatting.provider": "black"
+}
```

Comment:
Should this be checked in? Normally I think IDE configs are ignored, but there are potential benefits

Reply:
Let's ignore this until we get a consistent working one we like. I think I added it to .gitignore on one of the other branches.